packages/api/lambdas/process-s3-dead-letter-archive.js

```diff
@@ -47,13 +49,18 @@ async function processDeadLetterArchive({
           return deadLetterObject.Key;
```

Comment:
> Without an actual output returned from the Lambda, I think I was seeing the PG write fail on schema validation. Though now I feel like I need to double check this assertion

I don't quite understand how this would be the case when the PG write happens on line 48 in this try/catch block. Can you elaborate?

Reply:
So after re-testing this, the actual failure isn't the writing of the execution/granule/PDR records from `writeRecords`. It's in the async operation when we go to update the dead letter async operation with the `output` of the operation:

```
(node:1) UnhandledPromiseRejectionWarning: error: update "async_operations" set "status" = $1, "output" = $2, "updated_at" = to_timestamp(1629213124.621) where "id" = $3 - invalid input syntax for type json
```

The `output` field is the only JSON type field on an async operation.